directory_sso_api_client/backends.py

```diff
@@ -0,0 +1,40 @@
+import json
+import logging
+
+from requests.exceptions import RequestException
+
+from django.conf import settings
+
+from directory_sso_api_client import models, sso_api_client
+
+
+logger = logging.getLogger(__name__)
+
+
+class SSOUserBackend:
+    MESSAGE_INVALID_JSON = (
```

Comment:
Is this the right message?

It looks like we through it for any non 2xx response?
Would it be more helpful to log some details about the response that we got back?

Reply:
we do `logger.error(self.MESSAGE_SSO_UNREACHABLE, exc_info=True)`
`exc_info` means the execption info, (including the response object if present) will be sent to sentry.

Happy to change the message to some other string. "SSO did not return a 200 response", perhaps :)